packages/tabs/stories.tsx

```diff
@@ -126,8 +131,8 @@ storiesOf('Tabs Container', module)
                     item: tab,
                     key: tab,
                     style: {
-                      padding: !vertical && '10px 0',
```

Comment:
Styles that relied on the `&&` operator has been rewritten to rely on a ternary expression which evaluates to a `string` or `undefined` value instead. This is because TypeScript complains that `padding` cannot have a boolean value, which makes sense. For example, TypeScript says `padding` can have a value of type `string`, `number`, or `undefined`.

![Screen Shot 2019-10-07 at 11 12 33 AM](https://user-images.githubusercontent.com/1811365/66337533-2d3ac000-e8f4-11e9-9c97-e67675868a66.png)